constants/members.json

```diff
@@ -32,6 +32,18 @@
     {
       "name": "Haven",
       "id": "haven"
+    }, 
+    {
+      "name": "Infrastructure",
+      "id": "infrastructure"
+    },
+    {
+      "name": "Marrilac Place",
+      "id": "marillac-place"
```

Comment:
id will be `mp`. You'll have to replace all instances of `marillac-place` with `mp`. VSCode has find and replace  so it shouldn't be bad.